scripts/bootstrap.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dnf -y update
+dnf -y install python3-pip
+pip3 install --user Flask
+
+python3 /vagrant/scripts/app.py
```

Comment:
You can omit the python3 command. The shabang in app.py and the execute permissions should take care of it.